src/ssli_mbedtls.c

```diff
@@ -37,9 +37,13 @@
 #include <stdlib.h>
 #include <fcntl.h>
 
-#include <mbedtls/config.h>
 #include <mbedtls/version.h>
-#include <mbedtls/havege.h>
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+#include <mbedtls/psa_util.h>
+#else
+#include <mbedtls/ctr_drbg.h>
+#include <mbedtls/entropy.h>
+#endif
 #include <mbedtls/certs.h>
```

Comment:
```suggestion
#if MBEDTLS_VERSION_MAJOR < 3
#include <mbedtls/certs.h>
#endif
```

This got dropped with mbedtls3

Reply:
Yes, thank.  I incorporated your patch.